confmap/provider/googlesecretmanagerprovider/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+type: googlesecretmanagerprovider
+
+status:
+  class: provider
+  stability:
+    development: [provider]
+  codeowners:
+    active: [XuechunHou]
```

Comment:
Can you add @braydonk and me to the codeowners as well? Helps to have more folks on the team maintaining this component and be able to review changes.

Reply:
Done.